mis_builder_cash_flow/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 ADHOC SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'MIS Builder Cash Flow',
+    'version': '11.0.1.1.0',
+    'license': 'AGPL-3',
+    'author': 'ADHOC SA, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/mis-builder',
+    'depends': [
+        'mis_builder',
+    ],
+    'data': [
+        'security/mis_cash_flow_security.xml',
+        'report/mis_cash_flow_views.xml',
+        'views/mis_cash_flow_forecast_line_views.xml',
+        'data/mis_report_style.xml',
+        'data/mis_report.xml',
```

Comment:
Shouldn't this be demo instead of data?

Reply:
data with noupdate perhups, I will add noupdate